docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -73,25 +73,22 @@ A great documentation GitHub issue or PR includes:
 Verifying your documentation PR
 ================================
 
-If you make multiple changes to the documentation on ``ansible/ansible``, or add more than a line to it, before you open a pull request, please:
+If you make multiple changes to the documentation, or add more than a line to it, before you open a pull request, please:
 
 #. Check that your text follows our :ref:`style_guide`.
 #. Test your changes for rST errors.
 #. Build the page, and preferably the entire documentation site, locally.
 
 .. note::
 
-	The following sections apply to documentation sourced from the ``ansible/ansible`` repo and does not apply to documentation from an individual collection. See the collection README file for details on how to contribute to that collection.
+	The following sections apply to documentation sourced from the ``ansible/ansible-documentation`` repo and does not apply to documentation from an individual collection. See the collection README file for details on how to contribute to that collection.
 
 Setting up your environment to build documentation locally
 ----------------------------------------------------------
 
 To build documentation locally, ensure you have a working :ref:`development environment <environment_setup>`.
 
-To work with documentation on your local machine, you need to have python-3.9 or greater and install the `Ansible dependencies`_ and `documentation dependencies`_, which are listed in two files to make installation easier:
-
-.. _Ansible dependencies: https://github.com/ansible/ansible/blob/devel/requirements.txt
-.. _documentation dependencies: https://github.com/ansible/ansible/blob/devel/docs/docsite/requirements.txt
+To work with documentation on your local machine, you need to have python-3.9 or greater.
```

Comment:
For the `devel` branch this will soon be Python 3.10 (due to ansible-core dropping Python 3.9 support). While not all docs work involves running core code, it's probably better to list a version here that meets the core minimum requirement.

Reply:
Thanks, I've updated with a link to the support matrix because that is probably better maintained than here.